### DISCUSSION TITLE
export 'Particles' (imported as 'Particles') was not found in '@tsparticles/vue2' (possible exports: ParticlesComponent, default)

### DISCUSSION THREAD

**Discussion by ashim221** [tsparticles/vue2#37] [Q&A]
I have installed @tsparticles/vue2 using pnpm. however, I am getting this error 
'export 'Particles' (imported as 'Particles') was not found in '@tsparticles/vue2' (possible exports: ParticlesComponent, default)'

<img width="986" alt="image" src="https://github.com/tsparticles/vue2/assets/592423/1585ab96-6151-4d23-9c30-bde976201497">
<img width="1495" alt="image" src="https://github.com/tsparticles/vue2/assets/592423/9bfc590a-8a6f-47b5-9c17-90c0c0a5fef8">

**Comment by matteobruni** [comment]
Some code will help, please show your usage here.
